shell/migrations/0004_auto_20200225_1806.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.0.2 on 2020-02-25 12:36
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('shell', '0003_auto_20200209_1806'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='searches',
+            options={'verbose_name_plural': 'Searches'},
+        ),
+    ]
```

Comment:
When you're done with the dev on this PR, perhaps you could delete all these migrations and replace with a single one. It's inevitable that migrations will get messy over time, but at least initial one can be nice and simple ;)

Reply:
Yeah Sure. Once done I will replace them with a single one.

Reply:
I have deleted all the migrations. Replaced them with a single migration.